maintenance/uploadXmlDumpToSpectrumS3Bucket.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+require_once __DIR__ . '/../Maintenance.php';
```

Comment:
I think this is the reason why I can't run script, a simple `/../Maintenance.php` ->`/Maintenance.php`  should do the job, right?

Reply:
Or a move to `wikia` folder